bazel/third-party/BUILD.json-schema-validator.bazel

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
nit: please run buildfier on all of the build files to get rid of empty lines and indenting inconsistencies.

Reply:
Done